src/App/Pages/Accounts/TwoFactorPage.xaml

```diff
@@ -130,14 +130,47 @@
                     </StackLayout>
                 </StackLayout>
             </StackLayout>
-            <StackLayout Spacing="0" Padding="0" IsVisible="{Binding DuoMethod, Mode=OneWay}"
-                         VerticalOptions="FillAndExpand">
+            <StackLayout
+                Spacing="0"
+                Padding="0"
+                IsVisible="{Binding DuoMethod, Mode=OneWay}"
+                VerticalOptions="FillAndExpand">
+                <StackLayout
+                    StyleClass="box"
+                    Margin="0, 21, 0, 0"
+                    HorizontalOptions="FillAndExpand"
+                    VerticalOptions="FillAndExpand"
+                    Spacing="21"
+                    IsVisible="{Binding IsDuoFrameless}">
+                    <Label
+                        StyleClass="box-label-regular"
+                        Text="{Binding DuoFramelessLabel}"
+                        HorizontalOptions="StartAndExpand" />
+                    <StackLayout StyleClass="box-row, box-row-switch">
+                        <Label
+                            Text="{u:I18n RememberMe}"
+                            StyleClass="box-label-regular"
+                            HorizontalOptions="StartAndExpand" />
+                        <Switch
+                            IsToggled="{Binding Remember}"
+                            StyleClass="box-value"
+                            HorizontalOptions="End" />
+                    </StackLayout>
+                    <Button Text="{u:I18n LaunchDuo}"
+                        StyleClass="btn-primary"
+                        Command="{Binding DuoFramelessCommand}"
+                        AutomationId="DuoFramelessButton"/>
+                </StackLayout>
                 <controls:HybridWebView
                     x:Name="_duoWebView"
                     HorizontalOptions="FillAndExpand"
                     VerticalOptions="FillAndExpand"
-                    MinimumHeightRequest="400" />
-                <StackLayout StyleClass="box" VerticalOptions="End">
+                    MinimumHeightRequest="400"
+                    IsVisible="{Binding IsDuoFrameless, Converter={StaticResource inverseBool}}"/>
+                <StackLayout
+                    StyleClass="box"
+                    VerticalOptions="End"
+                    IsVisible="{Binding IsDuoFrameless, Converter={StaticResource inverseBool}}">
                     <StackLayout StyleClass="box-row, box-row-switch">
                         <Label
                             Text="{u:I18n RememberMe}"
```

Comment:
Thank i didn't put much thought to it because my brain saw the remember me in different places of the stack and immediately assumed there was no way around it. Silly me. 
Like you suggested used the visibility property and put all elements under the same stack. Added Margin to frameless items to keep their placement where it should be.